live-examples/css-examples/css/word-break.css

```diff
@@ -0,0 +1,5 @@
+#example-element {
+    width:10em;
```

Comment:
I would suggest making this 80% and also add `padding: 20px` to have some breathing room inside the box.